.circleci/config.yml

```diff
@@ -36,35 +19,64 @@ jobs:
             - dependency-cache-{{ checksum "package-lock.json" }}
 
       - run:
-          name: Install cqlsh
-          # We need to edit DEFAULT_CQLVER defined in the cqlsh script
-          # to connect to the server without --cqlversion command line option
+          name: Docker login
           command: |
-            curl -O https://bootstrap.pypa.io/get-pip.py
-            sudo python get-pip.py
-            sudo pip install cqlsh
-            sudo sed -i "s/^DEFAULT_CQLVER = .*/DEFAULT_CQLVER = '3.4.4'/" /usr/local/bin/cqlsh
+            echo ${GHCR_PAT} | docker login https://ghcr.io -u ${GHCR_USERNAME} --password-stdin
 
       - run:
-          name: Wait for Cassandra
-          # dockerize command is included in circleci/openjdk:8-jdk image
-          command: dockerize -wait tcp://127.0.0.1:9042 -timeout 1m
+          name: Update docker and docker compose
+          command: |
+            sudo apt-get purge docker-ce containerd.io
+            curl -fsSL https://get.docker.com -o get-docker.sh
+            sudo sh get-docker.sh
+            sudo curl -L "https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+            sudo chmod +x /usr/local/bin/docker-compose
 
       - run:
-          name: Install schema on Cassandra
-          command: cqlsh --execute "$SCHEMA_CONTENT" 127.0.0.1 9042
+          name: checkout scalar-samples code
```

Comment:
Could you update this text to explains why you are checking out this repository as it is probably not obvious ?